packages/store/docs/store-taxonomy.md

```diff
@@ -0,0 +1,67 @@
+# A Taxonomy of Stores
+
+Some dimensions on which our Stores differ. For each of these dimensions, in the headings below we indicate the dominant default choice with an asterisk (*). A given Store maker should document which non-default points in this design space it supports. The `makeScalarMapStore` function makes stores that default to all the default choices below, allowing only the non-default option to specify a schema, to create a schematized store.
+
+All stores are Far objects (a kind of remotable) that may be snapshotted to (or restored from) a corresponding pass-by-copy objects. MapStores snapshot into CopyMaps. SetStores snapshot into CopySets.
+
+# API Differences
+
+## SetStores vs MapStores*
+
+* SetStores store only keys (of type Key), where each key is key-unique, i.e., it is not keyEQ to any other key in the set.
+* MapStores store key value associations, where the keys are key-unique Keys, as with sets, and the associated values are Passable.
+
+## Weak vs Strong*
```

Comment:
Somewhere before this section, I think we need to talk about stores having four kinds of methods. The text from store/src/types.js is
```js
/**
 * Most store methods are in one of three categories
 *   * lookup methods (`has`,`get`)
 *   * update methods (`add`,`init`,`set`,`delete`,`addAll`)
 *   * query methods (`snapshot`,`keys`,`values`,`entries`,`getSize`)
 *   * query-update methods (`clear`)
```

Only then can we clearly say that Weak differs from Strong by omitting the query and query-update methods. See the verbiage in that types.js file.

Reply:
(Though the quoted comment above should say "four" rather than "three")

Reply:
I will wordsmith this.